### PR TITLE
flake8 conformance changes

### PR DESCRIPTION
**Pull request by dopstar**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,19 +47,25 @@
...
+    classifiers=[
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python",
+        "Operating System :: OS Independent",
+        "Topic :: System :: Networking :: Time Synchronization",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```
Comment (cf-natali) [inline setup.py:68]: Is it really necessary to list all the supported Python versions?
I don't really want to have to update this whenever a new Python version is released.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[bdist_wheel]
+universal = 1
+
+[flake8]
+max-line-length = 100
+inline-quotes = double
+
+[isort]
```
Comment (cf-natali) [inline setup.cfg:8]: I'm not convinced by putting isort configuration in this project - pep8 and pyflake are quite standard and add value, isort seems quite specific.

File: setup.py (hunk 1 above)
Comment (dopstar) [reply setup.py:68]: its not necessary but its useful

eg you could have this banner in the readme file as a result
Screenshot 2021-03-02 at 08 37 17

Comment (dopstar) [reply setup.py:68]: > I don't really want to have to update this whenever a new Python version is released.
new major python version is released once a year, the other releases are just bugfix releases which occur every +/- 3 months. these classifiers are for tracking major python versions only not every "bugfix" release and it is okay if these are not updated immediately.

File: setup.cfg (hunk 2 above)
Comment (dopstar) [reply setup.cfg:8]: it is used by[ flake8-isort ](https://pypi.org/project/flake8-isort/)

**Pull request closed without merging**